app/src/main/java/com/example/photostomusic/MainActivity.java

```diff
@@ -53,11 +53,14 @@ protected void onCreate(Bundle savedInstanceState) {
         bottomNavigationView.setOnNavigationItemSelectedListener(new BottomNavigationView.OnNavigationItemSelectedListener() {
             @Override
             public boolean onNavigationItemSelected(@NonNull MenuItem item) {
-                Fragment fragment;
+                Fragment fragment = null;
```

Comment:
were you noticing issues when you didn't init define fragment as null?

Reply:
Android Studio was complaining if it was not initialized, I did it to remove the alert